packages/fuel-indexer-database/postgres/src/lib.rs

```diff
@@ -3,11 +3,14 @@
 use fuel_indexer_database_types::*;
 use fuel_indexer_lib::utils::sha256_digest;
 use sqlx::{pool::PoolConnection, types::JsonValue, Postgres, Row};
+use std::time::{SystemTime, UNIX_EPOCH};
 use tracing::info;
 
 #[cfg(feature = "metrics")]
 use fuel_indexer_metrics::METRICS;
 
+const NONCE_EXPIRY: u64 = 3600; // 1 hour
```

Comment:
does this mean that a user requesting the service will have to sign again every hour?

Reply:
@rvmelkonian 
- No the nonce is used to authenticate, but is not the authentication token.
- After a user requests a nonce they have an hour to use it or lose it.